src/modules/common/Logger.ts

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+
+    Logging class for Stoa
+
+    Define the output format of Log and support console output
+    and file writing, depending on the 'NODE_ENV'.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import winston from "winston";
+import { format, transports } from "winston";
```

Comment:
```suggestion
import { format } from "winston";
```
`transports` are not used.
 You can replace `winston.transports` with `transports`.

If you use `winston.format`, I don't think you need to import.

Reply:
> Okay. Please check a few things.
> It is also necessary to add a log folder to the .gitignore

https://github.com/bpfkorea/stoa/blob/f5aced1a1b93e91ccac6c290d933826e1a2a43d7/.gitignore#L1-L3

It's already applied.

Reply:
I think you have a good tool.